src/utils/miscellaneous.ts

```diff
@@ -99,3 +99,18 @@ export function setStorageItem(key: string, data: unknown): void {
   const formattedData = JSON.stringify(data)
   return localStorage.setItem(key, formattedData)
 }
+
+// Used e.g inside middleware composition in useGlobalState
+export function compose(...funcs: Function[]): Function {
```

Comment:
redux has already a way of composing, right?
Both for middleware and for reducers

Reply:
we don't use `redux` but i'll check if it's shipped with `react`, very likely actually

Reply:
ok, I mean this two guys:

https://redux.js.org/api/applymiddleware

https://redux.js.org/api/combinereducers